proxy/api/ts/ts.h

```diff
@@ -2328,6 +2328,7 @@ tsapi TSReturnCode TSHttpTxnCacheLookupUrlGet(TSHttpTxn txnp, TSMBuffer bufp, TS
 tsapi TSReturnCode TSHttpTxnCacheLookupUrlSet(TSHttpTxn txnp, TSMBuffer bufp, TSMLoc obj);
 tsapi TSReturnCode TSHttpTxnPrivateSessionSet(TSHttpTxn txnp, int private_session);
 tsapi int TSHttpTxnBackgroundFillStarted(TSHttpTxn txnp);
+tsapi bool TSHttpTxnIsWebsocket(TSHttpTxn txnp);
```

Comment:
should be using "int" instead of "bool" to be consistent?
also we need an API review?